spec/popup/format-spec.js

```diff
@@ -0,0 +1,51 @@
+import format from '../../src/common/popup/utils/format.js';
+
+describe('format util', () => {
+  const ticket = {
+    id: 'BTC-042',
+    title: 'Weird Bug while eating cake',
+    type: 'bug'
+  };
+
+  describe('commit', () => {
+    it('formats', () => {
```

Comment:
How about `it('includes ticket id and title', …)` to make the description more readable?